### PR TITLE
Add Maxel.app v2.2

### PR DESCRIPTION
**Pull request by ibrokemypie**
*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked that the cask was not already refused in [closed issues].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-versions/pulls
[closed issues]: https://github.com/caskroom/homebrew-versions/issues?q=is%3Aissue+is%3Aclosed

Adds Maxel Beta to homebrew-versions

### CODE REVIEW

**Comment by reitermarkus** [comment]
https://github.com/caskroom/homebrew-cask/pull/27339 apparently was a trial version. Is the beta not a trial?

**Comment by ibrokemypie** [comment]
@reitermarkus no, the beta is not a trial

File: Casks/maxel-beta.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+cask 'maxel-beta' do
+  version '2.2,40'
+  sha256 'a7d756ad64312d4d9ab1056259d2a55ab7245fc652bd0686671f67c15a3bd416'
+
+  # hockeyapp.net/api/2/apps/eace0efdc2a9f246400a77168ca6850a was verified as official when first introduced to the cask
+  url "https://rink.hockeyapp.net/api/2/apps/eace0efdc2a9f246400a77168ca6850a/app_versions/#{version.after_comma}?format=zip"
+  appcast 'https://rink.hockeyapp.net/api/2/apps/eace0efdc2a9f246400a77168ca6850a',
+          checkpoint: 'eeb5538c76d225966ac31ca9b45a72ac0eac98be6bdc9fe04327cc1655a2db14'
+  name 'Maxel Beta'
+  homepage 'https://maxelapp.com/'
```
Comment (vitorgalvao) [inline Casks/maxel-beta.rb]: Where’s the homepage for the beta? We need to verify the download URL.

Comment (vitorgalvao) [inline Casks/maxel-beta.rb]: Where’s the homepage for the beta? We need to verify the download URL.

Comment (vitorgalvao) [inline Casks/maxel-beta.rb]: @ibrokemypie We need a page users can see the download link to verify it for themselves it is legitimate. [That’s the point of the comment](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/url.md#when-url-and-homepage-hostnames-differ-add-a-comment) (emphasis added):

> The comment doesn’t mean you should trust the source blindly, but **we only approve casks in which users can easily verify its authenticity with basic means**, such as checking the official homepage or public repository.

**Comment by miccal** [comment]
Closing as the `url` cannot be verified via the `homepage`.

**Pull request closed without merging**